### PR TITLE
Fixing null date error

### PR DESCRIPTION
**Pull request by NickCEBM**
Created `date_check` function to take care of instances where we have conditions that compare a date to a null

### CODE REVIEW

File: clinicaltrials/frontend/management/commands/load_data.py [hunk 1]
```diff
@@ -526,20 +531,22 @@ def convert_to_csv():
             td["pending_results"] = does_it_exist(soup.pending_results)
 
             td["pending_data"] = dict_or_none(parsed_json, [cs, "pending_results"])
+                                  
+            date_cond1 = td["available_completion_date"]
+                                  + relativedelta(years=1)
```
Comment (NickCEBM) [inline clinicaltrials/frontend/management/commands/load_data.py]: Wondering if I screwed up the indentation here and that's what is making the tests fail.

Comment (NickCEBM) [reply clinicaltrials/frontend/management/commands/load_data.py]: Nope, something else.

File: clinicaltrials/frontend/management/commands/load_data.py [hunk 2]
```diff
@@ -526,20 +531,18 @@ def convert_to_csv():
             td["pending_results"] = does_it_exist(soup.pending_results)
 
             td["pending_data"] = dict_or_none(parsed_json, [cs, "pending_results"])
+                                  
+            date_cond1 = td["available_completion_date"] + relativedelta(years=1) + timedelta(days=30)
+            
+            date_cond2 = td["available_completion_date"] + relativedelta(years=3) + timedelta(days=30)
```
Comment (sebbacon) [inline clinicaltrials/frontend/management/commands/load_data.py]: Did you just make this change for readability?

Comment (NickCEBM) [reply clinicaltrials/frontend/management/commands/load_data.py]: Sort of. Those conditions got moved around a bit from elsewhere in the code and I was worried the indents were not correct so I just moved them all to 1 line.

Seems that wasn't the case though.

**Comment by sebbacon** [comment]
Closing for #202

**Pull request closed without merging**